components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -303,6 +303,28 @@ Chef Infra Server lets you delete the existing clients one at a time. To delete
 
 {{< figure src="/images/automate/delete-a-client.png" alt="Delete a Client">}}
 
+### Nodes
+
+A [Node]({{< relref "nodes" >}}) is a communication between the Chef Infra Server and the Infra Clients. Nodes are used when an Infra Client asks for a policy from the Infra Chef Server.
```

Comment:
A node is a device that is managed by Chef Infra. During the Chef Infra Client, the client retrieved a policy from the server and converges it onto a node.